### PR TITLE
Wrap most paragraphs to 80 columns

### PR DESCRIPTION
**Pull request by strk**
Reduces patches size and makes them more readable

I think it should be done for all text files, only not
doing it yet here because I know of at least another
open PR that modifies CONTRIBUTING.md (and is hard to
review due to the loooong single-line being changed)

See https://github.com/go-gitea/gitea/pull/2394/files

### CODE REVIEW

**Comment by bkcsoft** [comment]
LGTM. And Agreed

**Comment by sapk** [comment]
Github offer rendered diff that display diff of the rendered mardown that is more strict on what is changed. For example for this PR has `No visible changes here.` : https://github.com/go-gitea/gitea/pull/2396/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8

Otherwise LGTM.

File: README.md
Comment (lunny) [inline README.md:22]: I think we have discussed this many times. I don't think this is a good idea.

**Review by lunny — CHANGES_REQUESTED** [review]

**Comment by strk** [comment]
On Sun, Aug 27, 2017 at 03:49:05PM -0700, Antoine GIRARD wrote:
> Github offer rendered diff that display diff of the rendered mardown that is more strict on what is changed.

I don't think it's good to restrict users/contributors options
to "use GitHub"... offline `git diff` and email notifications

@lunny there are 2 LGTM give by maintainer, saying "no" should
be justified by an high risk of project degradation, which is
not the case here. If you think 2 approvals are not enough
then please send a proposal to change project management policy.

File: README.md
Comment (bkcsoft) [reply README.md:22]: @lunny Any reason why we should _not_ do it? From my point of view we allow people to review using `git diff` and in GitHub nothing would change.

**Comment by lunny** [comment]
@strk Our policy is no need changed. A PR should have at least 2 LGTM for mergable. But that don't mean 2 LGTM PRs have to be merged. Even if there are 2 LGTM, if someone has different opinion, it should be discussed more.

File: README.md
Comment (lunny) [inline README.md:29]: the below 3 lines will be very strange after rendered.

**Review by lunny — CHANGES_REQUESTED** [review]

**Comment by lunny** [comment]
This is a story between `diffable` and `readable`. 80 means characters before rendering or after?

**Comment by strk** [comment]
80 characters *before* rendering

I believe markdown renders all on the same line unless an empty line
is encountered

File: README.md
Comment (bkcsoft) [reply README.md:29]: It will not :) https://github.com/strk/gitea/tree/wrap80#purpose

**Review by lunny — APPROVED** [review]

**Pull request merged**
